web/source/keyboards/kmwkeyboards.ts

```diff
@@ -799,7 +801,11 @@ namespace com.keyman {
      */    
     isCJK(k0?: any|KeyboardStub) { // I3363 (Build 301)
       let textProcessor = com.keyman.singleton.textProcessor;
-      var k = textProcessor.activeKeyboard, lg=''; 
+      var k;
```

Comment:
Starting around here, you'll see a number of methods that may have you asking "why not move this to the `Keyboard` wrapper?"

That's the plan for the follow-up PR; there were enough moving pieces in this one as it is.